micrometer-core/src/test/java/io/micrometer/core/instrument/binder/system/ProcessorMetricsTest.java

```diff
@@ -42,16 +39,7 @@ void cpuMetrics() {
 
     @Test
     void hotspotCpuMetrics() {
-        boolean jdk9 = false;
-        try {
-            Class.forName("java.lang.reflect.InaccessibleObjectException");
-            jdk9 = true;
-        } catch (ClassNotFoundException ignored) {
-        }
-
-        OperatingSystemMXBean osBean = ManagementFactory.getOperatingSystemMXBean();
-        assumeTrue(!jdk9);
-        assumeTrue(osBean.getClass().getName().equals("sun.management.OperatingSystemImpl"));
```

Comment:
This line didn't work on Java 9 because the implementation class is not accessible, but we need only check for the HotSpot-specific accessible interface in this test. This will work on Java 8 and 9.